bot/models.py

```diff
@@ -15,3 +15,14 @@ class User(models.Model):
 
     class Meta:
         db_table = 'user'
+
+
+class PennyChat(models.Model):
+    title = models.TextField()
+    description = models.TextField()
+    date = models.DateTimeField()
+    invitees = models.TextField()
+    channels = models.TextField()
+    view = models.TextField()
+    template_timestamp = models.TextField()
+    template_channel = models.TextField()
```

Comment:
Let's talk about adding a `status`. We did this for events at Eventbrite. The status was a number with a meaning - like 10=planning, 20=scheduled, 30=completed, 40=canceled - something like that.